addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -56,6 +56,7 @@ export class MassMailingHtmlField extends HtmlField {
         return {
             ...super.wysiwygOptions,
             onIframeUpdated: () => this.onIframeUpdated(),
+            onWysiwygChange: () => this.commitChanges({ shouldInline: false }),
```

Comment:
About the commit message: you should give examples of what operations of the snippetmenu can trigger this problem. This will be invaluable later on if somebody needs to try to reproduce the bug that this commit is fixing.

Reply:
done